lib/java_buildpack/framework/language_server_bin_exec_jdt.rb

```diff
@@ -35,32 +33,9 @@ def compile
         @logger.debug { "IPC VAL:#{ipcval}"}
       end
 
-      # (see JavaBuildpack::Component::BaseComponent#release)
-      def release
-
-        environment_variables = @droplet.environment_variables
-
-        myWorkdir = @configuration["env"]["workdir"]
-        environment_variables.add_environment_variable(ENV_PREFIX + "workdir", myWorkdir)
-        myExec = @configuration["env"]["exec"]
-        environment_variables.add_environment_variable(ENV_PREFIX + "exec", myExec)
-        portIn = @configuration["env"]["STDIN_PORT"]
-        environment_variables.add_environment_variable(ENV_PREFIX + "STDIN_PORT", portIn)
-        portOut = @configuration["env"]["STDOUT_PORT"]
-        environment_variables.add_environment_variable(ENV_PREFIX + "STDOUT_PORT", portOut)
-        
-        myIpc = @configuration["env"]["ipc"]
-        @logger.debug { "JDT Env vars IPC:#{myIpc}" }
-        myIpc.each do |key, value|
-          environment_variables.add_environment_variable(ENV_PREFIX + key, value)
-        end
-        
-      end
-
       protected
 
-      # (see JavaBuildpack::Component::VersionedDependencyComponent#supports?)
-      def supports?
+      def sup?
         @application.environment.key?(LSPSERVERS) && @application.environment[LSPSERVERS].split(',').include?("java")
```

Comment:
Since all implementations are identical we can move this line to LanguageServerBase and define languageId in each language class.

Reply:
done. please review again.